src/main/java/org/opentripplanner/routing/core/RoutingRequest.java

```diff
@@ -125,6 +125,14 @@
     /** Whether the trip must be wheelchair accessible. */
     public boolean wheelchairAccessible = false;
 
+    public int unknownWheelchairAccessAtStopPenalty = 600;
```

Comment:
Yes, these are arbitrary default values that can be overridden in the `router-config.json`.

Reply:
The property is called `noWheelchairAccessOnStreetReluctance` because here we take the length of the street into account when computing how much to increase the weight: Rather than adding a fixed value to the cost we take the length of the street and multiply with it the factor.

In other words, it's way worse to traverse a wheelchair-inaccessible street with a length of 1000 meters than one which is only 10 meters long.

Reply:
I will add comments to make this clearer.